### ISSUE TITLE
Better cross links in RST

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmt#3876]
Would be nice if the discussion about [grid registrations ](https://docs.generic-mapping-tools.org/dev/cookbook/options.html#option-nodereg)could cross link to [this entry](https://docs.generic-mapping-tools.org/dev/grdsample.html#t) when referencing the problem.  How to do that in RST?

**Comment by seisman** [comment]
Personally i prefer to have a subsection in the grid registration section, talking about the conversion between two registrations, then add a cross link in the grdsample -T option.

**Comment by PaulWessel** [comment]
Hm, yes, so move the figures from grdsample to a new subsection under the appendix and add a link from grdsample, right?

**Comment by seisman** [comment]
I mean moving the figure to the ["grid registration" section](https://docs.generic-mapping-tools.org/dev/cookbook/options.html#option-nodereg).

**Comment by PaulWessel** [comment]
If I make a new sub-section in options.rst with those figures

```
Switching registrations
^^^^^^^^^^^^^^^^^^^^^^^
```

How do add a link from grdsample to that place?

**Comment by seisman** [comment]
Add a label before the section title:
```
.. _switch-reg:

Switching registrations
^^^^^^^^^^^^^^^^^^^^^^^
```

then in grdsample, use 
```
:ref:`Link to the section <switch-reg>`
```

**Comment by seisman** [comment]
Closed by #3880.
